app/controlplane/pkg/biz/organization.go

```diff
@@ -144,47 +144,6 @@ func (uc *OrganizationUseCase) doCreate(ctx context.Context, name string, opts .
 	return org, nil
 }
 
-func (uc *OrganizationUseCase) Update(ctx context.Context, userID, orgID string, name *string) (*Organization, error) {
```

Comment:
you probably need to also update the `ent` schema in `data` to make it `inmmutable`

Reply:
Absolutely, I had the files no added to the stash...Submitting them, thanks!